central/role/validate.go

```diff
@@ -152,50 +151,10 @@ func ValidateSimpleAccessScope(scope *storage.SimpleAccessScope) error {
 		multiErr = multierror.Append(multiErr, errors.New("name field must be set"))
 	}
 
-	err := ValidateSimpleAccessScopeRules(scope.GetRules())
+	err := role.ValidateSimpleAccessScopeRules(scope.GetRules())
 	if err != nil {
 		multiErr = multierror.Append(multiErr, err)
 	}
```

Comment:
nit: I think we can rewrite `ValidateSimpleAccessScope` a little bit into `ValidateSimpleAccessScope`(has access scope id check and calls next function) and `ValidateSimpleAccessScopeProto`(checks name + rules).

That way we separate storage constraints(id) from proto struct constraints(name, rules)

Reply:
Made the change, but had to also leave two public methods: one for validating the `rules` proto, as this was required by the `role/service`, and the other validating the whole `simple access scope proto`.